openlibrary/macros/databarWork.html

```diff
@@ -1,5 +1,87 @@
 $def with (page, editions_page=False, render_times={})
```

Comment:
```suggestion
$def with (page, editions_page=False, render_times={}, title=None)
```
Let's pass the html for the title into the template.  This will reduce the amount of repeated code.